ci/version_var.sh

```diff
@@ -0,0 +1,21 @@
+
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+# Release Parameters
+BASE_VERSION=0.1.1
+IS_RELEASE=false
+
```

Comment:
In the edge-agent version of this file was a variable called ARCH. It appeared to be related to WASM, so I removed it from here and from line 21. Let me know if this wasn't the right thing to do.